using-conjoin/overview.md

```diff
@@ -4,18 +4,19 @@
+### **Automatic Coin Selection**
 
-Think of bitcoins as actual physical coins of different amounts in your pocket. When you spend them, you will combine the change, and you will get some new change back.
+Sending bitcoin with Wasabi wallet is simpler than ever. Just enter the address you want to send bitcoin to, then enter the amount of bitcoin or the dollar value you would like to send. Once you add labels to your bitcoin, the wallet will automatically select the best coin combinations. 
+
+These coins are also known as Unspent Transaction Outputs (UTXO). Think of bitcoins as actual physical coins of different amounts in your pocket. When you spend them, you will combine the coins and then get some new change returned to your wallet if you didn't have the exact amount. Avoiding change has the benefit of creating a smaller digital footprint on the blockchain as well as making your transactions more private. Wasabi Wallet will remind you of this during the Transaction Preview before sending the coins.
 
 ![Visual representation of Bitcoin UTXO's](../.gitbook/assets/utxo-preview.png)
 
 ### Labeling
 
 For every bitcoin transaction you send and receive, it is important to identify who is observing that transaction. The label serves the purpose of making sure that we are aware of who is sending us coins and who we are sending them to. Bitcoin transactions being public, we want to avoid as much as possible revealing our transaction history to anyone that might be keeping track of us.
 
-So for example, if you receive bitcoin as payment for a job you completed, you can write “Bull Bitcoin pay”.
-
-Now, if you want to send coins to someone else, you can use the same coin with the “Bull Bitcoin” label if you wish to sell them on the Bull Bitcoin platform and add the label “Bull Bitcoin sold”, since it is the same observer of that coin, ie. Bull Bitcoin. If however, you wish to purchase something and you do not want your employer to know about it, then you would use a different coin that they are not capable of observing the transaction history.
+So for example, if you receive bitcoin as payment for a job you completed for Bull Bitcoin, you can add the label “Bull Bitcoin” to that coin. Then if you want to send some bitcoin to Alice, you can use the same coin by adding the label "Alice". This coin would then have two labels, "Alice" and "Bull Bitcoin", since those are the two entities that know about this coin. If you wish to sell the coin on the Bull Bitcoin platform, just add the label “Bull Bitcoin” since it is the same observer of that coin. If however, you wish to purchase something and you do not want Bull Bitcoin to know about it, then you would use a different coin that they are not capable of observing the transaction history.
```

Comment:
I removed the last sentence because currently coin control is not implemented. It will be in the near future.

```suggestion
So for example, if you receive bitcoin from Bull Bitcoin as payment for a job you completed, you should add the label “Bull Bitcoin” to that coin. Then if you want to send some bitcoin to Alice, you can use the same coin by adding the label "Alice". The change coin would then have two labels, "Alice" and "Bull Bitcoin", since those are the two entities that know this change coin belongs to you. If you wish to sell the coin on the Bull Bitcoin platform, just add the label “Bull Bitcoin” as the recipient when sending.
```